### DISCUSSION TITLE
Help with .editorconfig (gear)

### DISCUSSION THREAD

**Discussion by lisbd** [dotnet/roslyn#74049] [Q&A]
Why can't I make screen 1 the same as screen 2, with the gear?
![diffgit](https://github.com/dotnet/roslyn/assets/173107240/504584da-3e1c-467f-abcd-4354e06f00b7)

**Comment by huoyaoyuan** [comment]
The question is related to VSCode and GitHub CodeSpace, not roslyn.
It's probably related to theme and plugin configuration.
